contentcuration/contentcuration/serializers.py

```diff
@@ -269,12 +269,10 @@ class Meta:
         fields = ('tag_name', 'channel', 'id')
 
 
-exercise_image_checksum_regex = re.compile("\!\[[^]]*\]\(\$" + exercises.IMG_PLACEHOLDER + "/([a-f0-9]{32})\.[\w]+\)")
-
-
 class AssessmentListSerializer(serializers.ListSerializer):
 
     def update(self, queryset, validated_data):
+        exercise_image_checksum_regex = re.compile(r"\!\[[^]]*\]\(\${placeholder}/([a-f0-9]{{32}})\.[\w]+\)".format(placeholder=exercises.IMG_PLACEHOLDER))
```

Comment:
Had to update this as the linter was throwing an error here. Still works the same though